openmetadata-docs/content/overview/releases/latest-release.md

```diff
@@ -4,157 +4,21 @@ slug: /overview/latest-release
-- Apache NiFi automates the flow of data between systems. OpenMetadata now supports a NiFi connector as the third new pipeline service on this release.
-
-## Lineage
-We’ve enhanced the performance of workflows by having a separate workflow for Lineage and Usage. By using two workflows for computing specific pieces of information, we can effectively filter down the queries to extract lineage.
-
-During table usage ingestion, the tables retrieved successfully will be cached, so that there is no need to repeat the same calls multiple times as many queries would be referencing the same tables.
-Usage queries have been optimized.
-A result limit has been added to Usage queries.
-
-## Global Settings
-The OpenMetadata Settings dropdown menu has been transformed into a single, centralized Settings page for added convenience in viewing all the available options. The Global Settings comprises setting options for Team Members, Access based on Roles and Policies, Services, Data Quality, Collaboration, Custom Attributes, and Integrations for webhooks and bots. Admins can view or update settings for various services like Slack, MS Teams, Webhooks, etc from the Global Settings page.
-
-
-## UI/UX Improvements
-The major UI UX improvements have been done around Roles and Policies and a Global Settings page. Quite a lot of tweaks have been made to the UI to improve the UX.
-
-When creating a new user or when a user is registering for the first time, the dropdown menu for Teams now displays an option to ‘Show All’ teams. Previously, we supported the display of only the first 10 teams. An option has also been provided to search and filter.
-UI improvements have been made on the Schema, Service, and Database details pages.
-Manage Tab has been replaced with the manage button on the UI.
-
-
-## Thanks to our Contributors
-We are thankful for the overwhelming feedback and support we received from our community. We are grateful to the following community members for their code contributions:
-
-- Pedro Sereno for capturing Metabase lineage with SQL lineage.
-- Francisco J. Jurado Moreno — for working on caching tables when ingesting database usage; optimizing Redshift usage query; Datalake connector performance; non-scheduled workflows; and to add a comment to tag all OpenMetadata-related queries.
-- Nihar Doshi — for helping with cleaning Atlas, Amundsen, and metadata_rest; and checking MSSQL with windows authentication.
-
-Thanks to Abcabhishek, Bleachzk, Bryson Edwards, Daniel, Fikrifikar, geoHeil, Laila Patel, Nicolas Parot Alvarez, Nilesh Khatri, Pauline Tolstova, Pedro Sereno, Preeti Jain, Sam Firke, Samuel Stuetz, Sidharth Reddy Pallerla, Taurus-Le, TheFu527, Tomislav Sabados, Trillhaa, Upen Bendre for creating issues in GitHub. Thanks to Sergey Stepanov, Zlgonzalez for your feedback and suggestions.
```

Comment:
do we have a list of contributors, if so, we can add those in the latest-release?

Reply:
@TeddyCr ^

Reply:
New Contributors
@07Himank made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9006
@dhruvinmaniar123 made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9147
@talalio made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9174
@karanh37 made their first contribution in https://github.com/open-metadata/OpenMetadata/pull/9307